### PR TITLE
Refactor [v105] FXIOS-4736: Adjust sharing settings

### PR DESCRIPTION
**Pull request by nbhasin2**

### CODE REVIEW

**Comment by nbhasin2** [comment]
This is ready for review but am just waiting for some answers from product side hence added a `do not merge` label.

**Review by yoanarios — APPROVED** [review]

**Comment by nbhasin2** [comment]
Got confirmation, we are good to merge.

cc @yoanarios

File: Client/AdjustHelper.swift [hunk 1]
```diff
@@ -34,10 +46,10 @@ final class AdjustHelper: FeatureFlaggable {
     /// Used to enable or disable Adjust SDK and it's features. We disable third party sharing by default.
```
Comment (yoanarios) [inline Client/AdjustHelper.swift:46]: update the comment because we are not disabling by default anymore

**Review by yoanarios — APPROVED** [review]

File: Client/AdjustHelper.swift (hunk 1 above)
Comment (nbhasin2) [reply Client/AdjustHelper.swift]: good catch, updated

**Pull request merged**